.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,42 @@
+name: Feature Request
+description: Create a proposal to request a feature
+title: "[Feature]: "
+labels: ["T:feature-request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+       ✰  Thanks for opening an issue! ✰    
+       Before smashing the submit button please review the template.
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: |
+        What are the user needs?
+        How could this solution fix the user facing problem? 
+      placeholder: Short, concise description of the proposed feature/changes to the repository
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem Definition
+      description: |
```

Comment:
https://github.com/cosmos/cosmos-sdk/blob/marko/17244/.github/ISSUE_TEMPLATE/feature-request.yml

Can we get this above? On dependabot repo it is above and it looks imho better. Not sure if it is the same attribute.

Reply:
what is above?

Reply:
![image](https://github.com/cosmos/cosmos-sdk/assets/29894366/069a71c5-fd68-42ec-8e40-01892ee5ca50)
vs
![image](https://github.com/cosmos/cosmos-sdk/assets/29894366/d0ff9a1b-a634-4c86-be37-f2a8c63facd6)

Sorry, I mean above the header.